src/pages/components/modal/accessibility.mdx

```diff
@@ -17,400 +17,109 @@ import {
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
-</Row>
+<Caption>
+  Keyboard navigation is constrained to the dialog. Tab cycles through the
+  components that take focus.
+</Caption>
 
-### Android screen reader tests
+</Column>
+</Row>
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListRow head>
-        <StructuredListCell head>Environment </StructuredListCell>
-        <StructuredListCell head>Results</StructuredListCell>
-      </StructuredListRow>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - Android version 9 with Talkback
-            <br />
-            - Chrome Version 73.0.3683.103 (Official Build) (64-bit)
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>Talkback test:</strong>
-            <OrderedList>
-              <ListItem>
-                Swipe left or right, or use the Alt+Arrow keys in quick nav mode
-                until you find the Launch modal button.
-              </ListItem>
-              <ListItem>
-                Double tap the button or press the Alt-Enter keys to launch the
-                modal. The close modal button is announces and has focus.
-              </ListItem>
-              <ListItem>
-                Swipe Left or Right to explore the modal. VO announces the text,
-                and focus on the Cancel and Save buttons.
-              </ListItem>
-              <ListItem>
-                Note: that navigation is not restricted to the modal only, after
-                the modal is open there is no way to know if there is an open
-                modal present, the user needs to remember it. Doubletap or press
-                Alt+Enter on the Close or Cancel buttons to close the modal.
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![Esc closes a modal, while Space and Enter keys activate buttons](images/modal-accessibility-2.png)
+
+<Caption>
+  The dialog is resolved by the user pressing Esc or activating a button.
+</Caption>
+
+</Column>
 </Row>
 
-### iOS screen reader tests
+### Focus handling
+
+When the dialog appears, the first item that gets focus depends on the type of
+dialog. Passive dialogs only contain a close button (X), so that takes focus.
+For dialogs which prompt for confirmation or user decision, the primary button
+takes focus (regardless of number of buttons). For destructive interactions, the
+“cancel” button takes focus, not the red danger/delete button. The tab order
+should proceed left and down from whichever item has focus then wrap back to the
+Close button (X).
```

Comment:
Close would not be capitalized as it's not a proper noun nor in an emphasis style as a label. 
```suggestion
close button (X).
```

Reply:
technically it is the accessible label name, but I take your point :)